support/windows-client/networking/tcp-ip-packet-loss-troubleshooting.md

```diff
@@ -0,0 +1,105 @@
+# Diagnosing Packet Loss on Windows
+
+## Overview
+
+Packet loss occurs whenever a network packet does not reach its intended destination. Some packet loss is normal, and a dropped packet does not automatically imply it is the cause of a higher-level networking issue. At other times, packet loss can cause reduced performance, and APIs or even applications to fail.
+
+## Capturing Packet Loss Diagnostics
+
+The first step when conducting a packet loss investigation is to capture [pktmon](https://learn.microsoft.com/en-us/windows-server/networking/technologies/pktmon/pktmon) traces, typically by using the pktmon command-line workflow. Pktmon is capable of capturing packet traces, attributing local packet loss to specific reasons and code locations, and collecting packet loss statistics. When combined with [Wireshark](https://www.wireshark.org/) analysis of protocol-level behavior, pktmon traces are sufficient to root cause many cases of packet loss. Pktmon will also be capable of collecting packet loss statistics within network cards, but as of early 2025, this feature is not available.
```

Comment:
I am not sure there is a lot of value mentioning a feature that is not available yet.
Do we know that it will ever be available, and it is just a matter of waiting for the release?

Reply:
Yes, I expect it's just a matter of waiting, but I don't know the exact timelines and they may be NDA.